packages/teams-js/src/internal/communication.ts

```diff
@@ -9,6 +9,7 @@ import { latestRuntimeApiVersion } from '../public/runtime';
 import { version } from '../public/version';
 import { GlobalVars } from './globalVars';
 import { callHandler } from './handlers';
+import HostToAppMessageDelayTelemetry from './hostToAppTelemetry';
```

Comment:
Noticing that this is defined as default export and it looks there are not so many existing default export.. any reasoning behind it?

Reply:
Not any particular reason. This module will only use this class and I don't think we will export additional things. So that's why I defaulted to use the default export